src/api/app/services/notification_service/web_channel.rb

```diff
@@ -1,27 +1,28 @@
 # This class ensures the :web channel will have only the most up-to-date notifications
 module NotificationService
   class WebChannel
-    ALLOWED_FINDERS = {
-      'BsRequest' => OutdatedNotificationsFinder::BsRequest,
-      'Comment' => OutdatedNotificationsFinder::Comment
-    }.freeze
+    ALLOWED_FINDERS = { 'BsRequest' => OutdatedNotificationsFinder::BsRequest,
+                        'Comment' => OutdatedNotificationsFinder::Comment }.freeze
 
     def initialize(subscription, event)
       @subscription = subscription
       @event = event
       @parameters_for_notification = @subscription.parameters_for_notification
                                                   .merge!(@event.parameters_for_notification)
```

Comment:
this first `merge!` can return `nil` no?  I think the first `merge` should be without `!`

Reply:
You are right, @vpereira. I've added some guards also. Please check it out.